config/puma.rb

```diff
@@ -1,2 +1,14 @@
 threads ENV['PUMA_MIN_THREADS'].to_i, ENV['PUMA_MAX_THREADS'].to_i
 workers ENV['PUMA_NUM_WORKERS'].to_i
+
+# Invoke Puma.stats from the main process
+if ENV['LOG_PUMA_STATS'].present?
```

Comment:
```suggestion
if ENV['LOG_PUMA_STATS']
```

Don't do `.present?` since that's an activesupport method and that's not loaded here. Do a `!= nil` if you really want to be careful but I think just checking if it exists is fine

Reply:
I must have tested in IRB with activesuport. I'm fine with taking off the .present?, I originally added it thinking people may get confused if `'false'` didn't work. Thanks!